ZenPacks/zenoss/ZenPackLib/lib/libexec/ZPLCommand.py

```diff
@@ -28,13 +27,28 @@
 from ..params.ZenPackSpecParams import ZenPackSpecParams
 from ..params.DeviceClassSpecParams import DeviceClassSpecParams
 from ..params.RRDTemplateSpecParams import RRDTemplateSpecParams
+from ..params.EventClassSpecParams import EventClassSpecParams
+from ..params.EventClassMappingSpecParams import EventClassMappingSpecParams
 from ..resources.templates import SETUP_PY
 from ..helpers.ZenPackLibLog import ZenPackLibLog, DEFAULTLOG
 from ..helpers.WarningLoader import WarningLoader
 from ..helpers.Dumper import Dumper
 from ..helpers.Loader import Loader
 from ..helpers.utils import optimize_yaml, load_yaml_single
 from ZenPacks.zenoss.ZenPackLib import zenpacklib
+unused(Globals)
+
+
+def stripped_yaml_dump(specparams, Dumper=Dumper):
+    outputfile = yaml.dump(specparams, Dumper=Dumper)
+
+    # tweak the yaml slightly.
+    outputfile = outputfile.replace("__builtin__.object", "object")
+    outputfile = re.sub(r"!!float '(\d+)'", r"\1", outputfile)
+    outputfile = re.sub(r"!ZenPackSpec", r"", outputfile)
+
+    print outputfile
+
```

Comment:
Are you still seeing this in the output?  I've tweaked the Dumper/Loader classes to get rid of this stuff (not the "__builtin__.object" but haven't seen that before).